pkg/graphql/engine_config_v2_test.go

```diff
@@ -277,6 +277,20 @@ func TestGraphqlFieldConfigurationsV2Generator_Generate(t *testing.T) {
 
 }
 
+func TestEngineV2Configuration_EnableSingleFlight(t *testing.T) {
```

Comment:
Sonar doesn't like this test name. I think the regex suggests underscores can only occur at the start of names? [It's a bit weird. ](https://sonarcloud.io/project/issues?id=TykTechnologies_graphql-go-tools&pullRequest=313&resolved=false&types=CODE_SMELL)

Reply:
I actually think the regex is wrong. I think the intention might actually be `^((_|[a-zA-Z0-9])+)$`.
`^(_|[a-zA-Z0-9]+)$` only allows an underscore if it's the only thing in the regex. :D